lib/katello_deploy/repositories.rb

```diff
@@ -22,8 +23,8 @@ def configure(koji_repos = false)
       if koji_repos
         setup_koji_repos(@os_version, @katello_version, foreman_version)
       else
-        bootstrap_katello(@katello_version, @os_version)
-        bootstrap_foreman(foreman_version, @os_version)
+        bootstrap_katello(@katello_version, @os_version) if @katello_version
+        bootstrap_foreman(@foreman_version, @os_version) if @foreman_version
```

Comment:
This doesn't seem to work for Foreman versions:

`
Running shell command: yum -y localinstall http://yum.theforeman.org/1.9/el7/x86_64/foreman-release.rpm
`

This should include "releases" when not nightly.  http://yum.theforeman.org/releases/1.9/el7/x86_64/foreman-release.rpm